app/src/main/java/com/github/gotify/picasso/PicassoHandler.java

```diff
@@ -0,0 +1,103 @@
+package com.github.gotify.picasso;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import com.github.gotify.R;
+import com.github.gotify.Settings;
+import com.github.gotify.Utils;
+import com.github.gotify.api.Callback;
+import com.github.gotify.api.CertUtils;
+import com.github.gotify.api.ClientFactory;
+import com.github.gotify.client.api.ApplicationApi;
+import com.github.gotify.log.Log;
+import com.github.gotify.messages.provider.MessageImageCombiner;
+import com.squareup.picasso.OkHttp3Downloader;
+import com.squareup.picasso.Picasso;
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import okhttp3.Cache;
+import okhttp3.OkHttpClient;
+
+/**
+ * Copyright (C) 2020 Felix Nüsse Created on 01.06.20 - 20:03
```

Comment:
Can we remove this? you still own the copyright regardless. I think this just bloats the source code, and is probably outdated from the start, because you copied some stuff from other files (like the makePicasso method).

Reply:
Sure! I have it in my template, so its autogenerated. Will dump it later!

Reply:
Excellent, the rest of the code looks good, will merge it after your change.